### PR TITLE
Release: 2023-09-26

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

- EPAS: updated [release notes table](https://github.com/EnterpriseDB/docs/pull/4840)  by @nidhibhammar 
- All products: Added [release date and updated title](https://github.com/EnterpriseDB/docs/pull/4843) to the latest versions by @dwicinas

### CODE REVIEW

File: product_docs/docs/epas/11/epas_rel_notes/index.mdx [hunk 1]
```diff
@@ -23,28 +23,29 @@ EDB Postgres Advanced Server 11 is built on the open source PostgreSQL 11. EDB P
...
-| [11.6.13](29_epas11.6.13_rel_notes.mdx)   | 19 Nov 2019| [11.6](https://www.postgresql.org/docs/11/release-11-6.html) | 
-| [11.5.12](31_epas11.5.12_rel_notes.mdx)   | 26 Aug 2019| [11.5](https://www.postgresql.org/docs/11/release-11-5.html) | 
-| [11.4.11](33_epas11.4.11_rel_notes.mdx)   | 25 Jun 2019| [11.4](https://www.postgresql.org/docs/11/release-11-4.html) | 
-| [11.3.10](35_epas11.3.10_rel_notes.mdx)   | 13 May 2019| [11.3](https://www.postgresql.org/docs/11/release-11-3.html) | 
-| [11.2.9](37_epas11.2.9_rel_notes.mdx)     | 22 Feb 2019| [11.2](https://www.postgresql.org/docs/11/release-11-2.html) | 
-| [11.1.7](39_epas11.1.7_rel_notes.mdx)     | 28 Nov 2018| [11.1](https://www.postgresql.org/docs/11/release-11-1.html) | 
\ No newline at end of file
+|                  Version                  | Release Date |                         Upstream Merge                         |
+| ----------------------------------------- | ------------ | -------------------------------------------------------------- |
+| [11.21.33](epas11_21_33_rel_notes.mdx)    | 25 Sep 2023  |                                                                |
```
Comment (djw-m) [inline product_docs/docs/epas/11/epas_rel_notes/index.mdx:28]: No upstream merge link?

Comment (nidhibhammar) [reply product_docs/docs/epas/11/epas_rel_notes/index.mdx:28]: The release annoucement doesn't say there is any kind of upstream merge: https://docs.google.com/document/d/1kTtbqoDD4D54zY2-Rhu50WR5Nfdptq96K5JwLj7DxDM/edit

Comment (djw-m) [reply product_docs/docs/epas/11/epas_rel_notes/index.mdx:28]: Ok, just bug fixes.... and the rel notes themselves were merged yesterday yes?

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**